pom.xml

```diff
@@ -30,15 +30,15 @@
   <properties>
     <argLine>-Dfile.encoding=${project.build.sourceEncoding}</argLine>
     <javadocAdditionalOptions>-html5 --allow-script-in-comments --no-module-directories</javadocAdditionalOptions>
-    <byte-buddy.version>1.12.1</byte-buddy.version>
+    <byte-buddy.version>1.12.3</byte-buddy.version>
     <hamcrest.version>2.2</hamcrest.version>
     <opentest4j.version>1.2.0</opentest4j.version>
-    <bnd.version>6.0.0</bnd.version>
+    <bnd.version>6.1.0</bnd.version>
     <cdg.pitest.version>0.1.0</cdg.pitest.version>
     <!-- Dependency versions overriding -->
     <junit.version>4.13.2</junit.version>
-    <junit-jupiter.version>5.8.1</junit-jupiter.version>
-    <mockito.version>4.0.0</mockito.version>
+    <junit-jupiter.version>5.8.2</junit-jupiter.version>
+    <mockito.version>4.1.0</mockito.version>
```

Comment:
it looks you are not up to date with the main branch, these changes are already in the main branch, can you rebase your branch on top of the latest main commit so that your PR only contain changes related to the issue you are solving.